chromium_src/chrome/browser/ui/views/location_bar/location_bar_view.h

```diff
@@ -6,8 +6,56 @@
 #ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_LOCATION_BAR_LOCATION_BAR_VIEW_H_
 #define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_LOCATION_BAR_LOCATION_BAR_VIEW_H_
 
+// Init and Update method names which we redefine below are too common, so
```

Comment:
actually for that reason we shouldn't override them this way. Maintaining a list of includes is way too fragile

Reply:
👍 I'll make those 2 back into patches then.

Reply:
Addressed by ba80f220f1dfc935d3670fb1d892e7fbbc0e5ee4